lib/src/provider/flutter_advanced_networkimage.dart

```diff
@@ -18,15 +18,15 @@ typedef Future<Uint8List?> _ImageProcessing(Uint8List data);
 class AdvancedNetworkImage extends ImageProvider<AdvancedNetworkImage> {
   AdvancedNetworkImage(
     this.url, {
-    this.scale: 1.0,
+    this.scale = 1.0,
     this.width,
     this.height,
     this.header,
-    this.useDiskCache: false,
```

Comment:
This is wrong, the correct style for default parameters is by using an equal sign. This is fixed in all places now.